doc/classes/AudioStreamPlayback.xml

```diff
@@ -31,28 +34,34 @@
 			<param index="1" name="rate_scale" type="float" />
 			<param index="2" name="frames" type="int" />
 			<description>
+				Override this method to customize how the audio stream is mixed while the playback is active.
+				[b]Note:[/b] It is not useful to override this method in GDScript or C#. Only GDExtension can take advantage of it.
 			</description>
 		</method>
 		<method name="_seek" qualifiers="virtual">
 			<return type="void" />
 			<param index="0" name="position" type="float" />
 			<description>
+				Override this method to customize what happens when seeking this audio stream at the given [param position], such as by calling [method AudioStreamPlayer.seek].
 			</description>
 		</method>
 		<method name="_start" qualifiers="virtual">
 			<return type="void" />
 			<param index="0" name="from_pos" type="float" />
 			<description>
+				Override this method to customize what happens when the playback starts at the given position, such as by calling [method AudioStreamPlayer.play].
 			</description>
 		</method>
 		<method name="_stop" qualifiers="virtual">
 			<return type="void" />
 			<description>
+				Override this method to customize what happens when the playback is stopped, such as by calling [method AudioStreamPlayer.stop].
 			</description>
 		</method>
 		<method name="_tag_used_streams" qualifiers="virtual">
 			<return type="void" />
 			<description>
+				Called if [method AudioServer.set_enable_tagging_used_audio_streams] has been set to [code]true[/code].
```

Comment:
```suggestion
				Called whenever the audio stream is mixed if [method AudioServer.set_enable_tagging_used_audio_streams] has been set to [code]true[/code] and the playback is active.
```

Include info about it being called after every mix step, and only if playback is active.